chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -949,6 +949,98 @@ Example
+
+Parameters
+**********
+
+A request is in the form:
+
+::
+
+    <Scheme>://<ServerName>/<ProtocoleVersion>/ch.swisstopo.terrain.3d/<Stylename>/<Time>/<TileMatrixSetId>/<Zoom>/<X>/<Y>.<FormatExtension>
+
+with the following parameters:
+
+===================    ==================================   ==========================================================================
+Parameter              Example                              Explanation
+===================    ==================================   ==========================================================================
+Scheme                 http or https                        The scheme type
+ServerName             terrain[0-5].geo.admin.ch
+Version                1.0.0                                The terrain service protocol version
+Layername              ch.swisstopo.terrain.3d (constant)   The name of the terrain layer. (only one terrain layer is available)
+StyleName              default                              mostly constant
+Time                   2015311201                           Date of tile generation in (ISO-8601).
+TileMatrixSet          4326 (constant)                      EPSG code for WGS84
+TileSetId              12                                   Zoom level (see below)
+X                      4309                                 The longitue index
+Y                      3111                                 The latitude index
+FormatExtension        terrain                              The file extension (a gzipped binary terrain file)
+===================    ==================================   ==========================================================================
+
+
+The *<TileMatrixSet>* **4326** is defined as follow::
+
+  MinX              5.013926957923385
+  MaxX              11.477436312994008
+  MinY              45.35600133779394
+  MaxY              48.27502358353741
+  TileWidth         256
+
+With the *<tileOrigin>* in the bottom left corner of the bounding box.
+
+=============================== ========= ========================================
+Resoultion [m/pixel at equator] Zoomlevel Availability
+=============================== ========= ========================================
+78271.80469                     0         [-180, -90, 90, 180]
+39135.90234                     1         [-180, -90, 90, 180]
+19567.95117                     2         [-180, -90, 90, 180]
+9783.975586                     3         [-180, -90, 90, 180]
+4891.987793                     4         [-180, -90, 90, 180]
+2445.993896                     5         [-180, -90, 90, 180]
+1222.996948                     6         [-180, -90, 90, 180]
+611.4984741                     7         [-180, -90, 90, 180]
+305.749237                      8         Ranges as defined in the layer.json file
```

Comment:
Having timestamps in the doc is not ideal. Theoretically, we could put this globally somehwere...